app/javascript/tests/patient/Patient.test.js

```diff
@@ -74,6 +75,17 @@ describe('Patient', () => {
     });
   });
 
+  it('Properly renders identification section when patient is a minor', () => {
+    const wrapper = shallow(<Patient details={mockPatient5} hoh={mockPatient1} collapse={true} edit_mode={false} jurisdiction_path="USA, State 1, County 2" />);
+    const section = wrapper.find('#identification');
+    expect(section.find('h4').text()).toEqual('Identification');
+    expect(section.find('.edit-link').exists()).toBeTruthy();
+    identificationFields.forEach((field, index) => {
+      expect(section.find('b').at(index).text()).toEqual(field + ':');
+    });
+    expect(section.find(IconMinor).exists()).toBeTruthy();
```

Comment:
the test before this should have `expect(section.find(IconMinor).exists()).toBeFalsy();`